### PR TITLE
Issue 88 Add new Role Apis

### PR DESCRIPTION
**Pull request by tungleduyxyz**
Related issue: https://github.com/killbill/killbill-client-ruby/issues/88
- Update an existing role permissions
- Retrieve Role definition

### CODE REVIEW

**Comment by tungleduyxyz** [comment]
Hi @reshmabidikar ,
Please help me to review this PR. Thanks

File: source/includes/_security.md [hunk 1]
```diff
@@ -708,6 +708,11 @@ RoleDefinition roleDefinition = securityApi.getRoleDefinition(role, requestOptio
 
 
 ```ruby
+
+role = 'testRole17thDec'
```
Comment (reshmabidikar) [inline source/includes/_security.md:712]: Nit: Change to `TestRole` to keep this consistent with the other code snippets

File: source/includes/_security.md [hunk 2]
```diff
@@ -893,6 +898,16 @@ securityApi.updateRoleDefinition(roleDefinition, requestOptions);
 ```
 
 ```ruby
+user = 'user'
+reason = 'reason'
+comment = 'comment'
+
+roleDef = KillBillClient::Model::RoleDefinition.new
+
+roleDef.role = 'testRole17thDec'
```
Comment (reshmabidikar) [inline source/includes/_security.md:907]: Nit: Change to `TestRole` to keep this consistent with other code snippets

**Review by reshmabidikar — APPROVED** [review]
Left minor comments, good to merge once these are addressed.

**Review by reshmabidikar — APPROVED** [review]

**Pull request merged**